pkg/trace/trace.go

```diff
@@ -29,6 +29,15 @@ type Info struct {
 	ReqInfo   RequestInfo  `json:"request"`
 	RespInfo  ResponseInfo `json:"response"`
 	CallStats CallStats    `json:"stats"`
+
+	POSIXStats POSIXStats `json:"posixInfo"`
+}
+
+// POSIXStats has information of a POSIX call
+type POSIXStats struct {
```

Comment:
StorageStats() or DiskCallStats instead of POSIX since backend is not always POSIX